src/components/editor-page/render-context/editor-to-renderer-communicator-context-provider.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import React, { createContext, useContext, useMemo } from 'react'
+import { EditorToRendererCommunicator } from '../../render-page/window-post-message-communicator/editor-to-renderer-communicator'
+
+const EditorToRendererCommunicatorContext = createContext<EditorToRendererCommunicator | undefined>(undefined)
+
+/**
+ * Provides the {@link EditorToRendererCommunicator editor to renderer iframe communicator} that is set by a {@link EditorToRendererCommunicatorContextProvider context provider}.
+ *
+ * @return the received communicator
+ * @throws Error if no communicator was received
```

Comment:
```suggestion
 * @throws {Error} if no communicator was received
```

I'm not sure if we do that elsewhere in this way, but the type should be in curly brackets see [here](https://esdoc.org/manual/tags.html#-return)